sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -13,20 +13,21 @@ import {
 import {
   ApplicationService,
   FormService,
-  StudentFileService,
   StudentService,
   WorkflowActionsService,
   ProgramYearService,
+  APPLICATION_DRAFT_NOT_FOUND,
+  ONLY_ONE_DRAFT_ERROR,
```

Comment:
can we rename it to MORE_THAN_ONE_APPLICATION_DRAFT_ERROR or SINGULAR_APPLICATION_DRAFT? I feel, ONLY_ONE_DRAFT_ERROR does not get me the understanding of what it is

Reply:
The value assigned to this const is "ONLY_ONE_APPLICATION_DRAFT_PER_STUDENT_ALLOWED", which is a bit more descriptive but also very long, that's why I kept the description long and the const name short.
I can change to MORE_THAN_ONE_APPLICATION_DRAFT_ERROR.

Reply:
Const renamed